From edf59cd912aa41c0c76056c72445bc6997fb8fa2 Mon Sep 17 00:00:00 2001
From: Johannes Lorenz <j.git@lorenz-ho.me>
Date: Sat, 19 Dec 2020 18:33:27 +0100
Subject: [PATCH 1674/1936] Update submodules to fix compiler warnings

The rtosc update is minor. The DPF update is 4 years ...
---
 DPF   | 2 +-
 rtosc | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/DPF b/DPF
index e0d8e14a..9243625a 160000
--- a/DPF
+++ b/DPF
@@ -1 +1 @@
-Subproject commit e0d8e14a14d407648768da7e34bcf31812245387
+Subproject commit 9243625aacb9fb8dd9fe4bd479b227149eb37959
diff --git a/rtosc b/rtosc
index a99d32ad..2138685a 160000
--- a/rtosc
+++ b/rtosc
@@ -1 +1 @@
-Subproject commit a99d32adafac16a7956e5636e5dd04b2a67ff54b
+Subproject commit 2138685a5b0a1d81eea9025360b7774c0b9fb12d
-- 
2.47.0

